packages/expo-updates/ios/EXUpdates/CodeSigning/EXUpdatesCertificateChain.swift

```diff
@@ -134,6 +147,18 @@ extension X509Certificate {
     
     return true
   }
+  
+  func expoProjectInformation() throws -> EXUpdatesExpoProjectInformation? {
```

Comment:
Made a task for validating this up the chain: ENG-4427. I think project info validation could happen in either this function or in the main chain validation.